src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -108,8 +110,14 @@ void tabulate() {
         // temporarily set config to single-seat so we can run sequential elections
         config.setNumberOfWinners(1);
         while (config.getSequentialWinners().size() < numWinners) {
-          Set<String> newWinnerSet = runTabulationForConfig(config);
-          assert (newWinnerSet.size() == 1);
+          // Read cast vote records and precinct IDs from CVR files
+          List<CastVoteRecord> castVoteRecords = parseCastVoteRecords(config, precinctIDs);
+          if (castVoteRecords == null) {
+            Logger.log(Level.SEVERE, "Aborting tabulation due to cvr errors!");
```

Comment:
```suggestion
            Logger.log(Level.SEVERE, "Aborting tabulation due to cast vote record errors!");
```

Reply:
[facepalm] this is what i get for working past my bedtime lol